### PR TITLE
The Butchers cleaver can now be stored in the belt slot

### PR DESCRIPTION
**Pull request by Arrow768**
Implements: https://forums.aurorastation.org/topic/11077-allow-the-butchers-cleaver-to-go-into-the-belt-slot/

### CODE REVIEW

**Comment by Conspiir** [comment]
That awkward moment when you say _not_ instead _now._

File: html/changelogs/arrow768-cleaver-belt.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - tweak: "The butcher's cleaver can not be stored in the belt slot."
```
Comment (Alberyk) [inline html/changelogs/arrow768-cleaver-belt.yml]: Can now.

Comment (Arrow768) [inline html/changelogs/arrow768-cleaver-belt.yml]:
```suggestion
  - tweak: "The butcher's cleaver can now be stored in the belt slot."
```

Comment (Arrow768) [reply html/changelogs/arrow768-cleaver-belt.yml]: Ultimate lazyness

**Pull request merged**